gen-client.sh

```diff
@@ -9,27 +9,61 @@ then
   exit 1
 fi
 
+normalize_version ()
+{
+  python3 -c '
+import sys
+from packaging.version import Version
+
+print(Version(sys.stdin.read()))
+'
+}
+
+generator_version ()
+{
+  python3 -c '
+import sys
+from packaging.version import Version
+
+language = sys.argv[1]
+core_version = Version(sys.stdin.read())
+
+if language.lower() == "python":
+    if core_version >= Version("3.70.dev"):
+        print("v4.3.1")
```

Comment:
Do you plan to update the generator to a higher version or wait until the next breaking change?

Reply:
Oh, I planned to add this if only when we really update the image...